src/path.js

```diff
@@ -0,0 +1,36 @@
+import { properties } from "./children.js";
+
+/**
+ * Find a path from an ancestor node to a descendant node.
+ * @param {object} ancestor the ancestor node to start the search from
+ * @param {object} descendant the descendant node to search for
+ * @returns {(string | number)[] | null} an array of keys to traverse to get from the ancestor to the descendant, or null if no path exists
```

Comment:
Should it return null or undefined if a path does not exist?

Reply:
`null`